_posts/2023-10-25-aws-lambda-linode-migrate.md

```diff
@@ -0,0 +1,383 @@
+---
+title: "Migrate AWS Lambda functions to Linode Kubernetes Engine with OpenFaaS"
```

Comment:
I'm thinking we take Linode / LKE out of the title so that this is more generic.

"How to migrate Python functions from AWS Lambda to OpenFaaS and Kubernetes"